Launcher/Classes/Constants.swift

```diff
@@ -38,9 +39,19 @@ enum Constants {
             static let changeLanguage = main.path(forResource: "change_sim_language", ofType: .bash)
             static let uniqueElements = main.path(forResource: "get_uniq_elements", ofType: .bash)
             static let simulators = main.path(forResource: "get_sim_list", ofType: .bash)
+            static let physicalDevices = main.path(forResource: "get_physical_device", ofType: .bash)
         }
         enum Ruby {
             static let helpers = main.path(forResource: "helpers", ofType: .ruby)
         }
     }
+    
+    enum DeviceType {
```

Comment:
Not sure if this should be here. I can imagine that this enum will have some more functionality in the future, and I feel it's not strictly just constants.